### ISSUE TITLE
Important context missing, Please Help

### ISSUE THREAD

**Issue by johnhhu2020** [MMBazel/Springboard-DataScienceTrack-Student#16]
Thank you for the txt file, I'm current doing same study on this topic with DataCamp course. 
There are something missing, I found in below test, Question 3 and Question 4 was missing. Do you have them? I mean if you have the access to DataCamp full context, please help me with it.  You can check my study on my GitHub page. 

> Bringing it all together: Star Wars profiling
> A list of 480 superheroes has been loaded into your session (called heroes) as well as a list of each hero's corresponding publisher (called publishers).

**Comment by johnhhu2020** [comment]
Thank you for  GitHub Springboard-DataScienceTrack-Student, I'm study on Efficient Python with DataCamp too

There are parts missing in your sharing, Chapter 2 Timing and profiling code, Question 3&4.
If you have full access to DataCamp context, please help me with it. You can check my GitHub page.

**Comment by MMBazel** [comment]
Hi John, 
I'd encourage you to try and answer the questions within Datacamp.  There's a good chance that the questions & answers I copied have changed in the past 4 years. But the questions are very reasonable and I went through the process of answering the questions without any help other than the hints in the Datacamp console. As far as I know, Springboard doesn't make students pass with a certain score, they just need to complete the Datacamp courses.
